### PR TITLE
Fix: Attachment - Hmm its not here page displayed on login with receipt attachment URL

### PR DESCRIPTION
**Pull request by dukenv0307**
### Details

The correct receipt attachment should be display after login with URL
### Fixed Issues

$ https://github.com/Expensify/App/issues/28568
PROPOSAL: https://github.com/Expensify/App/issues/28568#issuecomment-1742363882

### Tests

1. Create a money request with a receipt or create a scan request
2. Go to the transaction thread report
3. Click on the receipt image
4. Verify that the preview appears and the image display correctly
5. Copy the URL
6. Sign out and sign in again with this URL
7. Verify that after signing in the loading appears and then the image is displayed

- [x] Verify that no errors appear in the JS console

### QA Steps

1. Create a money request with receipt or create a scan request
2. Go to the transaction thread report
3. Click on the receipt image
4. Verify that the preview appears and the image display correctly
5. Copy the URL
6. Sign out and sign in again with this URL
7. Verify that after signing in the loading appears and then the image is displayed

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
        - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/Expensify/App/assets/129500732/1a6034e5-32e9-42e7-98ff-45a97de499e6

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/Expensify/App/assets/129500732/1b7f0769-b6c8-4150-ba3f-ba8b664063a2

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/Expensify/App/assets/129500732/2b9fab60-ae8e-4063-b16f-6574ea26f6f5

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/Expensify/App/assets/129500732/546d51a1-9c2d-4526-b22f-1a9b0681e9b0

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/Expensify/App/assets/129500732/e983227a-9397-49fa-9505-e20f43cee4a8

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/Expensify/App/assets/129500732/cdb87c0b-8080-438a-9fc9-5dafa18eb8b2

</details>

### CODE REVIEW

**Comment by situchan** [comment]
Sorry not sure how I missed this. On it now

**Comment by situchan** [comment]
Please pull main

File: src/components/Attachments/AttachmentCarousel/index.js [hunk 1]
```diff
@@ -153,6 +168,10 @@ function AttachmentCarousel({report, reportActions, source, onNavigate, setDownl
         [activeSource, canUseTouchScreen, setShouldShowArrows, shouldShowArrows],
     );
 
+    if (isLoadingReport || isLoadingReportAction) {
+        return <FullScreenLoadingIndicator />;
+    }
+
```
Comment (situchan) [inline src/components/Attachments/AttachmentCarousel/index.js:174]: We usually avoid duplicated code. What do you think about adding this logic in parent component - AttachmentModal?

File: src/components/Attachments/AttachmentCarousel/index.js [hunk 2]
```diff
@@ -38,6 +40,19 @@ function AttachmentCarousel({report, reportActions, source, onNavigate, setDownl
...
+    // For small screen, we don't call openReport API when we go to a sub report page by deeplink
+    // So we need to call openReport here for small screen
+    useEffect(() => {
+        if (!isSmallScreenWidth || (!_.isEmpty(report) && !_.isEmpty(reportActions))) {
+            return;
+        }
+        Report.openReport(report.reportID);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [isSmallScreenWidth, report.reportID]);
+
```
Comment (situchan) [inline src/components/Attachments/AttachmentCarousel/index.js:55]: We usually avoid duplicated code. What do you think about adding this logic in parent component - AttachmentModal?

**Comment by situchan** [comment]
@dukenv0307 bug still happening to me. Tested on desktop


https://github.com/Expensify/App/assets/108292595/00c65fe3-b004-4b4d-aecb-58fc9a4a2e17

**Comment by situchan** [comment]
@dukenv0307 I am not able to reproduce https://github.com/Expensify/App/issues/31105 what you're trying to fix.
Arrow buttons don't show for me, as expected.


https://github.com/Expensify/App/assets/108292595/c55c0d37-6fbf-4353-8c2b-b1a6327026e4

**Comment by mananjadhav** [comment]
@dukenv0307 Can you please merge the latest main and resolve the conflicts? The code change looks fine.

**Comment by situchan** [comment]
This PR is on hold for https://github.com/Expensify/App/pull/31467
@mananjadhav Maybe you meant to comment on https://github.com/Expensify/App/pull/31467?

**Comment by situchan** [comment]
[Old PR](https://github.com/Expensify/App/pull/31467) was reverted.
[New PR](https://github.com/Expensify/App/pull/32647) is still not merged yet

**Comment by shubham1206agra** [comment]
@situchan You can go ahead with the review here

**Comment by situchan** [comment]
We can wait a bit more until https://github.com/Expensify/App/pull/32647 reaches production as it needs to pass QA in staging

**Comment by situchan** [comment]
> Create a new screen to display receipt image and then prevent this bug as well.

@mountiny should we do this for consistency?
I see that all attachment preview modals are going to have their own routes in #28999 (@dukenv0307 you're working on that as well 😄)

**Comment by mountiny** [comment]
@situchan Seems like for consistency we should make this into its own route too

**Comment by situchan** [comment]
@dukenv0307 please continue now that hold is lifted

**Comment by situchan** [comment]
> We have a problem on main now: when we open this page or other pages like report avatar, profile avatar by deep link, `OpenApp` API isn't called so it shows loading page infinitely.

If so, please report in slack

**Comment by situchan** [comment]
in review

**Comment by situchan** [comment]
@dukenv0307 please merge main

File: src/ROUTES.ts [hunk 3]
```diff
@@ -429,6 +429,11 @@ const ROUTES = {
         getRoute: (transactionID: string) => `eReceipt/${transactionID}` as const,
     },
 
+    RECEIPT: {
+        route: 'receipt/:transactionID/:reportID',
```
Comment (situchan) [inline src/ROUTES.ts:433]: I think this route can be improved like `r/:reportID/receipt/:transactionID`

**Comment by situchan** [comment]
Regression: wrong receipt image (distance request)


https://github.com/Expensify/App/assets/108292595/9314d37f-c896-4966-970b-ec430b107813


Expected:
Screenshot 2024-03-03 at 11 30 09 PM

Actual:
Screenshot 2024-03-03 at 11 29 51 PM

**Comment by situchan** [comment]
> @situchan It works for me
> 
>  Screen.Recording.2024-03-04.at.11.39.57.mov

Logout on desktop app first

**Comment by situchan** [comment]
ok not reproducible anymore. I suspect it was recent regression from Onyx version bump

**Comment by situchan** [comment]
@dukenv0307 Tests step is now deprecated

**Comment by situchan** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>

**Comment by situchan** [comment]
I found weird bug while deleting receipt. Can you please check?

**Comment by situchan** [comment]
@mountiny I think this is backend bug. Delete receipt doesn't work (also happening on main)


https://github.com/Expensify/App/assets/108292595/32198411-09f3-420c-a4da-d574599d6040

Otherwise looks good

**Comment by mountiny** [comment]
@dukenv0307 you have got conflicts now

@situchan can you provide logs for the delete receipt call from networks tab, also what email is the account you are testing with so I can check the logs

**Comment by situchan** [comment]
> > @mountiny I think this is backend bug. Delete receipt doesn't work (also happening on main)
> 
> @situchan This is a know bug here #34427, it's expected that we cannot delete the receipt of fail scan request.

ok, let's just resolve conflict

File: src/components/ReportActionItem/MoneyRequestView.tsx [hunk 4]
```diff
@@ -261,9 +261,9 @@ function MoneyRequestView({
                                     thumbnail={receiptURIs?.thumbnail}
                                     image={receiptURIs?.image}
                                     isLocalFile={receiptURIs?.isLocalFile}
-                                    filename={receiptURIs?.filename}
                                     transaction={transaction}
                                     enablePreviewModal
+                                    filename={receiptURIs?.filename}
```
Comment (situchan) [inline src/components/ReportActionItem/MoneyRequestView.tsx:266]: here

File: src/components/ReportActionItem/ReportActionItemImage.tsx [hunk 5]
```diff
@@ -37,11 +37,11 @@ type ReportActionItemImageProps = {
...
     isLocalFile?: boolean;
 
-    /** Filename of attachment */
-    filename?: string;
-
     /** Whether there are other images displayed in the same parent container */
     isSingleImage?: boolean;
+
+    /** Filename of attachment */
+    filename?: string;
```
Comment (situchan) [inline src/components/ReportActionItem/ReportActionItemImage.tsx:44]: here

File: src/components/ReportActionItem/ReportActionItemImage.tsx [hunk 6]
```diff
@@ -50,7 +50,7 @@ type ReportActionItemImageProps = {
  * and optional preview modal as well.
  */
 
-function ReportActionItemImage({thumbnail, image, enablePreviewModal = false, transaction, isLocalFile = false, filename, isSingleImage = true}: ReportActionItemImageProps) {
+function ReportActionItemImage({thumbnail, image, enablePreviewModal = false, transaction, isLocalFile = false, isSingleImage = true, filename = ''}: ReportActionItemImageProps) {
```
Comment (situchan) [inline src/components/ReportActionItem/ReportActionItemImage.tsx:53]: here

File: src/components/ReportActionItem/ReportActionItemImage.tsx [hunk 7]
```diff
@@ -109,9 +109,11 @@ function ReportActionItemImage({thumbnail, image, enablePreviewModal = false, tr
                 {({report}) => (
                     <PressableWithoutFocus
                         style={[styles.w100, styles.h100, styles.noOutline as ViewStyle]}
-                        onPress={() => Navigation.navigate(ROUTES.TRANSACTION_RECEIPT.getRoute(report?.reportID ?? '', transaction?.transactionID ?? ''))}
-                        accessibilityLabel={translate('accessibilityHints.viewAttachment')}
+                        onPress={() => {
+                            Navigation.navigate(ROUTES.RECEIPT.getRoute(transaction?.transactionID ?? '', report?.reportID ?? ''));
+                        }}
                         accessibilityRole={CONST.ROLE.BUTTON}
+                        accessibilityLabel={translate('accessibilityHints.viewAttachment')}
```
Comment (situchan) [inline src/components/ReportActionItem/ReportActionItemImage.tsx:116]: here

**Review by situchan — COMMENTED** [review]
Please undo unnecessary changes.

File: src/components/ReportActionItem/ReportActionItemImage.tsx [hunk 8]
```diff
@@ -109,9 +109,11 @@ function ReportActionItemImage({thumbnail, image, enablePreviewModal = false, tr
                 {({report}) => (
                     <PressableWithoutFocus
                         style={[styles.w100, styles.h100, styles.noOutline as ViewStyle]}
-                        onPress={() => Navigation.navigate(ROUTES.TRANSACTION_RECEIPT.getRoute(report?.reportID ?? '', transaction?.transactionID ?? ''))}
```
Comment (situchan) [inline src/components/ReportActionItem/ReportActionItemImage.tsx:112]: Looks like receipt modal is already using separate route?

Comment (situchan) [reply src/components/ReportActionItem/ReportActionItemImage.tsx:112]: Oh same work was being done at the same time in other place. We can close this PR

**Comment by situchan** [comment]
> @situchan @dukenv0307 to confirm, are these two issues completely same? is there nothing to address after the other PR was merged?

The only difference is `isLoading`, `shouldShowNotFoundPage` check logic and whether to import `parentReportAction` from utils or Onyx directly in Receipt component.
Other than this, all changes look exactly the same.

**Comment by mountiny** [comment]
@dukenv0307 but the original issue this PR was created for:

1. Open the app
2. Click on plus and raise manual request money request with any amount and any user
3. Open the request money page by clicking twice on IOU message
4. edit URL and add '/edit/receipt' in it
5. Upload any image and after upload, open it
6. Copy the URL
7. Logout and visit the copied URL
8. Login and observe that it displays 'Hmm its not here' page

Is not repro?

**Comment by situchan** [comment]
@mountiny here's [context](https://github.com/Expensify/App/pull/30561#issuecomment-1864071890)

Screenshot 2024-03-11 at 10 30 49 PM

**Comment by mountiny** [comment]
So can you please close this PR if not needed anymore? @dukenv0307

**Pull request closed without merging**